tests/User/AutoLoginMiddlewareTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
```

Comment:
`declare(strict_types=1);` is required. Please check other places

Reply:
Updated in the last commit

Reply:
Not resolved